cmake/projects/Boost/hunter.cmake

```diff
@@ -148,6 +148,17 @@ hunter_add_version(
     e6bb97b5109c7c15ea459cf2b1a9d52cbf88a89e
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    Boost
+    VERSION
+    "1.71.0"
```

Comment:
Please match the version to the release. In this case, `1.71.0-p0`.

Reply:
Thanks for pointing out, I will update the version to 1.71.0-p0